components/panelbar/overview.md

```diff
@@ -12,18 +12,8 @@ position: 0
 
 The <a href="https://www.telerik.com/blazor-ui/panelbar" target="_blank">Blazor PanelBar component</a> displays [data]({%slug panelbar-data-binding-overview%}) ([flat]({%slug panelbar-data-binding-flat%}) or [hierarchical]({%slug panelbar-data-binding-hierarchical%})) in an accordion type structure. In addition to built-in [navigation capabilities]({%slug panelbar-navigation%}), you can navigate through the items and their children, define [templates]({%slug panelbar-templates%}), render text and [icons/images]({%slug panelbar-icons%}), and respond to [events]({%slug panelbar-events%}).
 
-
-
-#### In this article:
-
-* [Basics](#basics)
-
-* [Elements of a PanelBar Item](#elements-of-a-panelbar-item)
-
-
-## Basics
-
-To use a Telerik PanelBar for Blazor:
+ 
+## Creating Blazor PanelBar
```

Comment:
Point 3 is not necessary in this case. We can only mention that such customization is possible, if the model class is using non-default property names. We can mention 2-3 default property names in point 2 or 3.

Probably we should describe the default property names first, and then add a step for binding the PanelBar to a collection of such items.